Back_end/routes/spotifyTokenRoutes.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router = express.Router();
+const axios = require('axios');
+require('dotenv').load();
+var SpotifyWebApi = require('spotify-web-api-node');
+
+router.use(express.json());
+
+var clientId = "b56e28cbf84e4c38aec748a3f8891a29";
+var clientSecret = "517016abe18342c193482decf3c669e1";
```

Comment:
Uploading "secret" keys for an API or service can allow malicious people to use them to ring up charges against your account. You should generate a new set of keys, then *not* upload the "secret" key again, opting to expose it through something like a key store or an environment variable depending on where you're deploying to.

Reply:
Hi Vera, I realized I made that mistake right after I did the pull request. I have immediately reset our secret key.